CHANGES.md

```diff
@@ -63,8 +63,9 @@
 ## New Features / Improvements
 
 * Local packages can now be used as dependencies in the requirements.txt file, rather
-  than requiring them to be passed separately via the `--extra_package` option.
-  ([#23684](https://github.com/apache/beam/pull/23684))
+  than requiring them to be passed separately via the `--extra_package` option
+  (Python) ([#23684](https://github.com/apache/beam/pull/23684)).
```

Comment:
Technically not part of this PR, but could you add `(java)` to the breaking change below?

Reply:
Done and caught another one where the issue link wasn't linkified.